### PR TITLE
Fix some small problems with set window rect changes

### PR DESCRIPTION
**Pull request by jgraham**
* Restore handling both missing and null values for parameters.

* Fix some grammar errors.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver/pull/1832.html" title="Last updated on Jul 17, 2024, 1:55 PM UTC (ee975a4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver/1832/bb939cc...ee975a4.html" title="Last updated on Jul 17, 2024, 1:55 PM UTC (ee975a4)">Diff</a>

### CODE REVIEW

**Comment by whimboo** [comment]
This is a follow-up fix for https://github.com/w3c/webdriver/pull/1830 and contributes to a final solution for issues as covered on https://github.com/w3c/webdriver/issues/1826.

File: index.html [hunk 1]
```diff
@@ -4046,28 +4046,39 @@ <h4><dfn>Set Window Rect</dfn></h4>
   <a>getting a property</a> named "<code>width</code>"
   from <var>parameters</var></a>.
 
+ <li><p>If <var>width</var> is <a>undefined</a> let <var>width</var>
```
Comment (OrKoN) [inline index.html:4049]:
```suggestion
 <li><p>If <var>width</var> is <a>undefined</a>, let <var>width</var>
```

File: index.html [hunk 2]
```diff
@@ -4046,28 +4046,39 @@ <h4><dfn>Set Window Rect</dfn></h4>
...
+  be null.
+
  <li><p>Let <var>height</var> be the result of
   <a>getting a property</a> named "<code>height</code>"
   from <var>parameters</var>.
 
+ <li><p>If <var>height</var> is <a>undefined</a> let <var>height</var>
```
Comment (OrKoN) [inline index.html:4056]:
```suggestion
 <li><p>If <var>height</var> is <a>undefined</a>, let <var>height</var>
```

File: index.html [hunk 3]
```diff
@@ -4046,28 +4046,39 @@ <h4><dfn>Set Window Rect</dfn></h4>
...
+  be null.
+
  <li><p>Let <var>x</var> be the result of <a>getting a property</a>
   named "<code>x</code>" from <var>parameters</var>.
 
+ <li><p>If <var>x</var> is <a>undefined</a> let <var>x</var> be null.
```
Comment (OrKoN) [inline index.html:4062]:
```suggestion
 <li><p>If <var>x</var> is <a>undefined</a>, let <var>x</var> be null.
```

File: index.html [hunk 4]
```diff
@@ -4046,28 +4046,39 @@ <h4><dfn>Set Window Rect</dfn></h4>
...
+
  <li><p>Let <var>y</var> be the result of <a>getting a property</a>
   named "<code>y</code>" from <var>parameters</var>.
 
- <li><p>If <var>width</var> or <var>height</var> is neither <a>undefined</a>
-  nor a <a>Number</a> from 0 to 2<sup>31</sup> − 1,
-  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+ <li><p>If <var>y</var> is <a>undefined</a> let <var>y</var> be null.
```
Comment (OrKoN) [inline index.html:4067]:
```suggestion
 <li><p>If <var>y</var> is <a>undefined</a>, let <var>y</var> be null.
```

**Review by OrKoN — APPROVED** [review]
it looked like commas are needed in the if statements

**Review by whimboo — APPROVED** [review]
Good find. I was already wondering about the removal but didn't notice that we missed those cases. Beside what @OrKoN already mentioned it looks good to me. Thanks.

**Pull request merged**